pinot-plugins/pinot-input-format/pinot-avro-base/pom.xml

```diff
@@ -50,5 +50,9 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Can you add this dependency into pinot-spi? It should stay with log4j-slf4j-impl